src/api/views.py

```diff
@@ -41,6 +44,22 @@ def get(self, request, format=None):
         serializer = UserSerializer(user, many=False)
         return Response(serializer.data)
 
+    def post(self, request, format=None):
+        data = {
```

Comment:
These parameters need to be in the function docstring, so that swagger will show input fields when test-driving. Check out how it's done with the RepoForUser post method.